ocdaction/tasks/models.py

```diff
@@ -17,4 +17,39 @@ class Task(models.Model):
     # TODO add a task anxiety score once score model is created
 
     def __str__(self):
-        return self.taskname
\ No newline at end of file
+        return self.taskname
+
+class AnxietyScore(models.Model):
+    ZERO = '0'
```

Comment:
Can we rename these to be a little more descriptive like SCORE_ZERO so it is clear it is a score